src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -172,6 +172,10 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\ExtensionClientName\\Generated"
     },
+    "FlatArray": {
```

Comment:
Let's make this project name less specific. FlattenedParameters?

Reply:
Done